dev/NavigationView/NavigationBackButton.xaml

```diff
@@ -14,10 +16,15 @@
         <Setter Property="VerticalContentAlignment" Value="Center" />
         <Setter Property="UseSystemFocusVisuals" Value="{StaticResource UseSystemFocusVisuals}" />
         <Setter Property="Content" Value="&#xE72B;" />
+        <Setter Property="Margin"  Value="4,2" />
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource OverlayCornerRadius}" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="Button">
-                    <Grid x:Name="RootGrid" Background="{TemplateBinding Background}">
+                    <Grid x:Name="RootGrid"
+                        Background="{TemplateBinding Background}"
+                        contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
+                        contract7NotPresent:CornerRadius="{ThemeResource OverlayCornerRadius}">
```

Comment:
I don't understand the logic here. If contract 7 is present OR not present set CornerRadius?

Reply:
If contract7Present, it is template bound, so a developer is able to set their own CornerRadius through a public property API. Otherwise, it pulls from ThemeResources. Both evaluate to the same default value, one just makes it settable.

Reply:
[CornerRadius](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.cornerradius?view=winrt-19041)property on Control was added only with contract 7. So if we are running on or above contract7, we can template bind to CornerRadius property of the control and have a setter on the template to set the CornerRadius property to the OverlayCornerRadius theme resource. If not then Control.CornerRadius property is not available and we just directly set it using the ThemeResource. [Grid.CornerRadius ](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.grid.cornerradius?view=winrt-19041) exists as far as we care, so we don't have to worry about down-level there.